examples/mcp23Sxx_event_detect_interrupt.py

```diff
@@ -7,13 +7,13 @@
 
 import board
 import busio
-from digitalio import Direction, Pull
+from digitalio import DigitalInOut, Direction, Pull
 from RPi import GPIO
 from adafruit_mcp230xx.mcp23s17 import MCP23S17
 
 # Initialize the SPI bus:
 spi = busio.SPI(board.SCK_1, MOSI=board.MOSI_1, MISO=board.MISO_1)
-cs = digitalio.DigitalInOut(board.CS0)
```

Comment:
I'm surprised this worked beforehand. It's my understanding that it shouldn't. But regardless the change looks correct.